packages/patternfly-4/react-core/src/components/ApplicationLauncher/ApplicationLauncherItem.tsx

```diff
@@ -25,29 +27,71 @@ export interface ApplicationLauncherItemProps {
    * } />
    */
   component?: React.ReactNode;
+  /** Flag indicating if the item is favorited */
+  isFavorite?: boolean;
+  /** Aria label text for favoritable button when favorited */
+  ariaIsFavorite?: string;
```

Comment:
Maybe append `Label` to the end of these so it is more obvious that it is a label and not a boolean.